request_a_govuk_domain/request/utils.py

```diff
@@ -367,6 +367,6 @@ def phase_header_text(_request):
     """
     phase = os.getenv("ENVIRONMENT")
     return {
-        "PHASE_HEADER": "Prototype" if phase != "dev" else "Beta",
+        "PHASE_HEADER": "Prototype" if phase != "prod" else "Beta",
         "PHASE_CONTENT": "This is not a full service. It is not production code and you might experience problems.",
```

Comment:
I think phase_text also needs to change on prod?
https://sgs-ddt-01-96c924f9e494.herokuapp.com/registration/current/phase-banners